components/k8s-model-server/README.md

```diff
@@ -162,3 +162,107 @@ You can learn more about [updating a Deployment](https://kubernetes.io/docs/conc
 [scaling a Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#scaling-a-deployment), and 
 [Pod Resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) in the 
 Kubernetes documentation.
+
+
+
+### Use the served model
+
+The [inception-client](./inception-client) directory contains a Python script you can use to make a call against the deployed model.
+
+This script is intended to be run externally to the kubernetes cluster as a demonstration that the inception model is correctly being served.
+You can run the script either directly from a Python2 environment or in a Docker container.
+
+#### Setup
+
+You will require the external IP for the inception service as well as the port it is being hosted on. The inception service should be
+listed under the value you used for the `MODEL_NAME` parameter in the ksonnet component. You can find this information using
+```commandline
+kubectl get services
+NAME         TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)			 AGE
+$MODEL_NAME  LoadBalancer   <INTERNAL IP>   <SERVICE IP>     <CLUSTER PORT>:<NODE PORT>  <TIME SINCE DEPLOYMENT>
+```
+
+We will feed the `<SERVICE IP>` and `<CLUSTER PORT>` to the labelling script. We will use it to label the following image of a
```

Comment:
What is the cluster port? Where is set?

Reply:
Maybe I am using incorrect terminology - I meant the host port on the kubernetes cluster.

It is set here: https://github.com/google/kubeflow/blob/master/kubeflow/tf-serving/tf-serving.libsonnet#L27

Reply:
I think the term would just be SERVICE port.

I would prefer the guide had an instruction like this
kubectl port-forward `kubectl get pods --selector=name=airflow -o jsonpath='{.items[*].metadata.name}'` 8180:8080

That showed people how to setup port forwarding and then people connect via the service.

Or better yet would be to run it from within cluster e.g via a Jupyter Notebook. From within the cluster they can the service name as a DNS name.

Or they could use 
kubectl run 
to start up a shell inside a Docker container running within the cluster in which case they can again use the ServiceName

Reply:
Sounds good - this is related to https://github.com/google/kubeflow/issues/121 and I will update these instructions as part of that work.

Reply:
I went with `kuberentes run`

Reply:
The reason I didn't do it from the jupyterhub is that it only offers a Python3 kernel and there isn't a clean way to add the tensorflow-serving-api package there. I could directly use the protos I suppose, but I decided to go with something that required less explanation.